src/Npgsql/NpgsqlConnector.cs

```diff
@@ -147,6 +147,11 @@ sealed partial class NpgsqlConnector : IDisposable
         /// </summary>
         internal readonly Dictionary<string, string> PostgresParameters;
 
+        /// <summary>
+        /// Holds all run-time parameters in raw, binary format for efficient handling without allocations.
+        /// </summary>
+        readonly List<(byte[] Name, byte[] Value)> _rawParameters = new List<(byte[] Name, byte[] Value)>();
```

Comment:
Are field names required on the left side of the expression?

Reply:
Not any more - they were a bit useful before, but in your new code they're not used at all. Removing.